### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by kemigabocatherine**
**In this project;**

- I created a site based on a design by Cindy Shin.
- Created two web pages :page_facing_up:  Home page and About page.
- Each page has separate html, CSS, and javascript files
- Each page has designs for both the mobile :iphone: and desktop view. :computer: 
- I also created a dynamic menu for both pages
**Click** [here](https://www.loom.com/share/6a4b3691675c43afb6b26b3c86cbe5fe) **to watch my video**

### CODE REVIEW

File: About.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
```
Comment (gemmen29) [inline About.html:14]: Kindly try to match all heading for sections(main program, speaker, partner) in the home and section logo in about with the Behance design.
| Your design | Behance design |
| --- | --- |
| image | image |
| image | image |
| image | image |
| image | image |

- Kindly decrease those header `font-size`.
- Kindly make the header perfectly centered in the container.
- Please make the red line smaller and in the middle of the text as in the Behance design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      <div class="part2">
+        <div>
+          <img class="partner3" src="images/futuristic.png" alt="futuristic">
+        </div>
+        <div>
+          <img class="partner4" src="images/socl-observer.png" alt="observer">
+        </div>
+      </div>
+    </div>
+  </section>
```
Comment (gemmen29) [inline index.html]: Kindly try to match the partner section in the mobile version to match the Behance design.
| Your design | Behance design |
| --- | --- |
| image | image |
- Kindly make all logos appear in the middle of the container.
- Kindly add this section to the about page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  </div>
+  
+  <!--Program-->
+  <section class="program" id="program">
+    <div>
+      <h3 class="program-heading">
+        Main Program
+      </h3>
+    </div>
+    <section class="program-list">
```
Comment (gemmen29) [inline index.html:83]: Kindly try to match each box in the main program section with the Behance design in the mobile version.
| Your design | Behance design |
| --- | --- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+          <div class="program-title">
+            <h4 class="card-title">Transform project</h4>
+          </div>
+          <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+        </div>
+      </div>      
+    </section>
+    <button class="all-programs" type="button">
+        See All Programs
+      </button>
```
Comment (gemmen29) [inline index.html]: Kindly change the design of the `see all program` button to match the Behance design in the desktop version.
| Your design | Behance design |
| --- | --- |
| image | image |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+            <li class="menu-link" ><a href="#program">Program</a>
+            </li>
+            <li class="menu-link"><a href="#leaders">Leaders</a>
+            </li>
+            <li class="menu-link"><a href="#partners">Partners</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
```
Comment (gemmen29) [inline index.html]: - Kindly add the black header for the desktop version to match the Behance design in the desktop version.
- Kindly make the `ul` tag that contains `li` tags centered vertically as the space on the bottom is bigger than the space on the top.

| Your design | Behance design |
| --- | --- |
| image | image |

File: index.html (hunk 2 above)
Comment (gemmen29) [inline index.html]: Kindly try to match the partner section in the desktop version to match the Behance design.
| Your design | Behance design | 
| --- | --- |
| image | image |

File: About.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+  </section>
+  <footer id="footer">
+    <div class="footer-image">
+      <img class="footer-logo" src="images/logo3.png" alt="logo">
+    </div>
+    <div class="info">
+      <p>@socl School of Community Leadership</p>
+      <p class="sub"><sub>Some rights are reserved</sub></p>
+    </div>
+  </footer>
```
Comment (gemmen29) [inline About.html]: - Kindly try to match the footer with the Behance design in the mobile version:
   - Kindly change the ` background-color ` to be white.
   - Kindly make the logo and paragraph centered vertically in the container.
   - Kindly change the `font-size` of the paragraph to be smaller.
- Kindly Kindly make the logo and paragraph centered vertically in the container in the desktop version.

| Your design | Behance design |
| --- | --- |
| image | image |

File: README.md [hunk 7]
```diff
@@ -1 +1,37 @@
-# Capstone-1
\ No newline at end of file
+# Capstone Project
+
+School of Community Leadership-First capstone project.
+
```
Comment (gemmen29) [inline README.md:4]: Kindly add a screenshot of your website in your README file to make the README file more professional.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @kemigabocatherine, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good files organisation ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 7 above)
Comment (mateo951) [reply README.md:4]: Addressed

File: About.html (hunk 6 above)
Comment (mateo951) [reply About.html]: Addressed!

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html]: Addressed!

File: index.html (hunk 5 above)
Comment (mateo951) [reply index.html]: Addressed!

File: index.html (hunk 4 above)
Comment (mateo951) [reply index.html]: Addressed!

File: index.html (hunk 3 above)
Comment (mateo951) [reply index.html:83]: Addressed!

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html]: Addressed!

**Review by mateo951 — APPROVED** [review]
### Hi @kemigabocatherine, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it  🎉

### Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**